name.abuchen.portfolio.ui/src/name/abuchen/portfolio/ui/views/SecuritiesChart.java

```diff
@@ -52,12 +55,13 @@
 {
     private enum ChartDetails
     {
-        CLOSING(Messages.LabelChartDetailClosingIndicator), //
-        INVESTMENT(Messages.LabelChartDetailInvestments), //
-        EVENTS(Messages.LabelChartDetailEvents), //
-        SMA50(Messages.LabelChartDetailSMA50), //
-        SMA200(Messages.LabelChartDetailSMA200), //
-        DIVIDENDS(Messages.LabelChartDetailDividends);
+        typeCLOSING(Messages.LabelChartDetailClosingIndicator + "\\-----\\"), //
```

Comment:
Ist wahrscheinlich nicht ganz so problematisch, aber durch das Umbenennen der enum Werte gehen die aktuellen Einstellungen der Benutzer verloren. Sie müssten also noch mal neu entscheiden, sich den SMA50 einblenden zu lassen. Das liegt daran, dass der *Name* der enum in den Settings gespeichert wird.